src/main/kotlin/kweb/Element.kt

```diff
@@ -26,7 +26,7 @@ open class Element(
         val creator: ElementCreator<*>?,
         @Volatile open var jsExpression: String,
```

Comment:
Can we remove this now that id is required?

Reply:
I think so. I'll have to do that on my branch where I'm working on function caching though.